bartop-api/src/utils/errorCreator.js

```diff
@@ -0,0 +1,6 @@
+module.exports.notFound = id => {
+  const newError = new Error();
+  newError.name = 'ResourceNotFoundError';
+  newError.message = `Resource with ID:${id} does not exist.`;
```

Comment:
Nit picky but would you like to add a space between `ID:` and `${id}`

Reply:
Also if you wanted you could pass the `message` in the constructor but I kinda like that the `name` is assigned before the `message`

Reply:
did that in the other PR actually